### ISSUE TITLE
NVSE wont open at all

### ISSUE THREAD

**Issue by BeanBoy14** [xNVSE/NVSE#193]

**Comment by BeanBoy14** [comment]
it gives me the gambryo error and i tried the normal fix and it does nothing
the only thing i have is a text file called falloutnverror.txt

Fallout: New Vegas
Computer Name:
Initializing Renderer...
Create D3D9 instance...SUCCESSFUL

NiDX9Renderer::Initialize> Attempting to create rendering device

NiDX9Renderer::Initialize> Could not create device of desired type - trying next-lower device type

NiDX9Renderer::Initialize> Attempting to create rendering device

NiDX9Renderer::Initialize> Could not create device of desired type - trying next-lower device type

NiDX9Renderer::Initialize> Attempting to create rendering device

NiDX9Renderer::Initialize> Could not create device of desired type - trying next-lower device type

NiDX9Renderer::Initialize> Attempting to create rendering device

NiDX9Renderer::Initialize> Could not create hardware device of any type - FAILING

**Comment by lStewieAl** [comment]
Please send rendererinfo.txt

**Comment by BeanBoy14** [comment]
sorry for the late response but its empty for me

**Comment by BeanBoy14** [comment]
sorry my bad i had to uninstall nvse and open the game for it to work 

Renderer Device Information:
	NVIDIA GeForce RTX 3050 Laptop GPU
	nvldumd.dll
	RenderPath         		: BSSM_SV_2_A
	PSversion          		: 300
	VSversion          		: 300
	VStarget           		: vs_2_0
	PStarget           		: ps_2_a
	PS2xtarget         		: ps_2_a
	maxPS20inst        		: 512
	3.0 Support        		: yes
	3.0 Lighting       		: yes
	Nonpowerof2textures		: yes
	FP16ARGB blending  		: yes
	FP16ARGB filtering 		: yes
	High dynamic range 		: yes
	Bloom lighting     		: no
	Refraction         		: yes
	2.0 hairw           		: yes
	SLI mode           		: no
	Water shader       		: yes
	Water reflections  		: yes
	Water displacement 		: yes
	Water high res     		: no
	Multisample Type   		: 4
	Transparency MS    		: yes
	Shader Package     		: 13
	HW Thread Count    		: 12

**Comment by BeanBoy14** [comment]
i get the same problem when i install it on vortex

**Comment by BeanBoy14** [comment]
> Please send rendererinfo.txt from %USERPROFILE%/Documents/My Games/FalloutNV

is there any way to fix my problem?

**Comment by AVeryUncreativeUsername** [comment]
Yes! Do not lose hope young one! This guide will.....guide you through all you need to do to get playing FNV. 

https://vivanewvegas.moddinglinked.com/

**Comment by BeanBoy14** [comment]
thank you

**Comment by migrgh** [comment]
If you enable ASLR on windows 11 fallout new vegas unable to run even without NVSE.

**Comment by Demorome** [comment]
Someone installed the 4GB Patcher on nexus and apparently that fixed it for them, could try that.
